packages/p2p/src/components/orders/order-table/order-table-content.jsx

```diff
@@ -13,29 +13,29 @@ import OrderInfo from 'Components/orders/order-info';
 import { height_constants } from 'Utils/height_constants';
 
 const OrderTableContent = ({ showDetails, is_active }) => {
-    const {
-        changeTab,
-        is_active_tab,
-        list_item_limit,
-        order_offset,
-        order_table_type,
-        orders,
-        setOrders,
-        setOrderOffset,
-    } = React.useContext(Dp2pContext);
-    const [is_mounted, setIsMounted] = React.useState(false);
-    const [has_more_items_to_load, setHasMoreItemsToLoad] = React.useState(false);
+    const { changeTab, handleNotifications, list_item_limit, order_table_type, setOrders } = React.useContext(
+        Dp2pContext
+    );
+
     const [api_error_message, setApiErrorMessage] = React.useState('');
+    const [has_more_items_to_load, setHasMoreItemsToLoad] = React.useState(false);
+    const is_active_tab = React.useRef(true);
     const [is_loading, setIsLoading] = React.useState(true);
+    const [is_mounted, setIsMounted] = React.useState(false);
+    const order_offset = React.useRef(0);
+    const [orders, setOrdersState] = React.useState([]);
 
     React.useEffect(() => {
         setIsMounted(true);
         return () => setIsMounted(false);
     }, []);
 
     React.useEffect(() => {
+        setIsLoading(true);
```

Comment:
Should this be inside the `if (is_mounted)` below? Else the state could be updated when the component is not mounted.

Reply:
Oops, my bad, I was playing around with some states. Done!